probe-rs/build.rs

```diff
@@ -20,6 +20,17 @@ fn main() {
 
     let mut files = vec![];
     visit_dirs(Path::new("targets"), &mut files).unwrap();
+
+    // Check if there are any additional targets to generate for
+    match env::var("PROBE_RS_ADDITIONAL_TARGETS_DIR") {
```

Comment:
```suggestion
    match env::var("PROBE_RS_TARGETS_DIR") {
```

I like the shorter form more :)

Reply:
Totally agree that shorter is better. Will fix as soon as I can